ovh/resource_dedicated_server_bringyourownimage_test.go

```diff
@@ -0,0 +1,63 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDedicatedServerBringYourOwnImage(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheckCredentials(t)
+			testAccPreCheckDedicatedServer(t)
+			testAccPreCheckByoiImageUrl(t)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDedicatedServerBringYourOwnImageConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"ovh_dedicated_server_bringyourownimage.os_install", "function", "reinstallServer"),
+					resource.TestCheckResourceAttr(
+						"ovh_dedicated_server_bringyourownimage.os_install", "status", "done"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDedicatedServerBringYourOwnImageConfig() string {
+	dedicated_server := os.Getenv("OVH_DEDICATED_SERVER")
+	url := os.Getenv("OVH_BYOI_IMAGE_URL")
```

Comment:
```suggestion
	url := os.Getenv("OVH_DEDICATED_SERVER_BYOI_IMAGE_URL")
```

Reply:
An hardcoded variable might be enough.
If not, I think we need to keep the coherence of env variable name.